README.md

```diff
@@ -48,14 +52,25 @@ If no color is found, the last color in the list is used as a fallback.
 To use a single color regardless of the value just use a single list entry with any value to always trigger
 the fallback.
 
-Colors may be configured for both gauges at once or for each gauge individualy. In the latter case,
-colors for then inner gauge are shaded by 25%.
+
+Colors may be configured for both gauges at once or for each gauge individualy. By default,
+colors for then inner gauge are shaded by 25% (see option _shadeInner_).
 
 The list is automatically sorted so you don't need to do that in your config - but I recommend it anyways.
 
-## Example
+### min and max config
+
+There is 2 ways to config the minimum and maximum value of the gauge:
+
+1. **Configuration in the "Config" part** - The vaule for outer and inner gauge will be the same 
+2. **Configuration in outer and inner separately** - outer and inner will have different values , as you set
+    (you must to config the outer **and** the inner gauge, otherwise the the  gauge will have the default values.)
```

Comment:
"you have to configure" and there's 2 times "the" and an extra space after "otherwise"

And actually, that wouldn't be the case if you would implement my change request to prefer individual config over the global config. If one of the gauges has no own config, it would fallback to the global config.